### PR TITLE
Redesign

### PR DESCRIPTION
**Pull request by OlegMoshkovich**
PR includes redesign(restyle) of the following components:
- Buttons
- Dialog
- SnackBar
- Search
- Open Dialog
- Share Dialog
- About Dialog
- NavTree
- Item Properties
- Item Properties refactor
- SideDrawer
- Replacement of Icons
- Theme adjustments

### CODE REVIEW

**Comment by OlegMoshkovich** [comment]
@pablo-mayrgundter PTAL

File: src/Components/IssuesControl.test.jsx [hunk 1]
```diff
@@ -33,24 +33,23 @@ describe('IssueControl', () => {
...
+  const {getByText} = render(<ShareMock><Issues/></ShareMock>)
+  await act(() => {
+    result.current.setSelectedIssueId(testIssueId)
+  })
+  await act(() => {
+    result.current.setIssues(MOCK_ISSUES)
+  })
+  await act(() => {
+    result.current.setComments(MOCK_COMMENTS)
   })
```
[the remark this answers is not in the document]
Comment (OlegMoshkovich) [reply src/Components/IssuesControl.test.jsx:49]: I am not sure how it happened but the failure of these test case is related to the bug that made a way into Production.
At the moment when the issue card is clicked, the view changes to the individual issue  but the comments do not display.

[the remark this answers is not in the document]
Comment (OlegMoshkovich) [reply src/Components/IssuesControl.test.jsx:49]: image

[the remark this answers is not in the document]
Comment (OlegMoshkovich) [reply src/Components/IssuesControl.test.jsx:49]: the bug is filed here #400

File: src/Components/CutPlaneMenu.jsx [hunk 2]
```diff
@@ -1,15 +1,14 @@
 import React, {useState, useEffect} from 'react'
```
[the remark this answers is not in the document]
Comment (OlegMoshkovich) [reply src/Components/CutPlaneMenu.jsx:1]: I just resorted them - took a long time actually - following the comment you made yesterday.

**Comment by pablo-mayrgundter** [comment]
So, to perhaps make this more objective.  Can we agree that the usual typography does go from heavy/large to light/normal for the transition from organizational headers to text?  I see on every UI I have open in my browser:

GitHub:
image

Matrix:
image

GMail:
image

Discord:
image

File: src/Components/CutPlaneMenu.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (pablo-mayrgundter) [reply src/Components/CutPlaneMenu.jsx:1]: Ok, will check.. thought I saw some out of order

**Comment by OlegMoshkovich** [comment]
@pablo-mayrgundter 
Of course, of course ... but maybe


Like here when I look at the list I would rather pay attention to the 'wand', rather then the 'name'

image

Or even here
'together' is so much more interesting to me than 'name'
image



But if both of you think it should be otherwise then be it

**Comment by pablo-mayrgundter** [comment]
Yeah, I see that.

I think we're all trying to get to the same point of helping the user understand what they're looking at.  I guess it's about accommodating the expected paradigm.  Label and value are important and there's a normal way of indicating which is which.

I know you were working on using theme more.  Would be nice to compare side-by-side or see what normal weighting looks like and compare with screens of the current l&f

**Comment by OlegMoshkovich** [comment]
Changed typography and reordered the imports.
@pablo-mayrgundter PTAL

**Review by pablo-mayrgundter — APPROVED** [review]

**Pull request merged**